test/definitions/infra-agent/debians/ubuntu21x04hirsute-infra copy.json

```diff
@@ -0,0 +1,34 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+        "id": "host1",
+        "display_name": "Ubuntu21.04HirsuteInfraHost",
```

Comment:
Small nit: the name of this file is `ubuntu21x04hirsute-infra copy.json` - maybe rename it to remove the ` copy` part

Reply:
Good catch, TY